.travis.yml

```diff
@@ -17,7 +17,7 @@ python:
     - 2.7
 
 addons:
-    postgresql: "9.4"
+    postgresql: "9.2"
```

Comment:
If only I can remember **why** I changed this to 9.2 for this PR. Grr.... Time sucks.